release.md

```diff
@@ -65,7 +65,7 @@ The following instructions assume **`v0.X.Y` is the new version** and **v0.x.y**
     fullversion = "v0.x.y"
     version = "v0.x"
     githubbranch = "0.x.y"
-    docsbranch = "0.x"
+    docsbranch = "v0.x"
```

Comment:
Are we really sure about this change (`v`...)?

Reply:
Yes, it's the branch name. The current ones are:
- https://github.com/falcosecurity/falco-website/tree/v0.28
- https://github.com/falcosecurity/falco-website/tree/v0.27
- https://github.com/falcosecurity/falco-website/tree/v0.26

The example in the documentation missed the `v` prefix, so I fixed it.

Reply:
Wanted to double-check. Tnx